src/amber/cli/commands/main_command.cr

```diff
@@ -0,0 +1,38 @@
+module Amber::CLI
```

Comment:
From an organization perspective it's cleaner to leave the main command outside of commands since it's a different namespace, than the things inside of commands.